### ISSUE TITLE
v0.2.1 - Link Not found.

### ISSUE THREAD

**Issue by cremesk** [michael-lazar/jetforce#14]
Hello,

i have update my jetforce Server from Version 0.2.0 to 0.2.1.

after the update my links are not working anymore.

log v0.2.0 (works)
```
Mar 31 09:21:25 xxx jetforce: "gemini://xxx" 31 "gemini://xxx/" 23                                                                                                        
Mar 31 09:21:25 xxx jetforce: "gemini://xxx/" 20 "text/gemini" 949                                                                                                             
Mar 31 09:21:27 xxx jetforce: "gemini://xxx/~foo/" 20 "text/gemini" 303
```

log v0.2.1 (not work)
```
Mar 31 09:18:27 xxx jetforce: "gemini://xxx" 31 "gemini://xxx/" 23                                                                                                          
Mar 31 09:18:27 xxx jetforce: "gemini://xxx/" 20 "text/gemini" 949                                                                                                              
Mar 31 09:18:30 xxx jetforce: "gemini://xxx/~foo/" 51 "Not Found" 14
```

Thanks for Help
Best regards, ~creme

**Comment by michael-lazar** [comment]
Thanks for the report! It looks like there was a regression, and I was too tired to properly smoke test the release last night.

I have fixed the bug and bumped the version to 0.2.2.

https://github.com/michael-lazar/jetforce/releases/tag/v0.2.2

https://github.com/michael-lazar/jetforce/commit/ff2832f034fba8bac5c7a08de34af01ccd00ea62

**Comment by cremesk** [comment]
Yeah thanks for checking. it works now in v0.2.2! =)
